pages/blog/creating-consistency-announcing-asyncapi-spectral-together.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Creating Consistency: Announcing AsyncAPI & Spectral – Together!"
+date: 2022-04-13T6:20:59+02:00
```

Comment:
since there are just some final editorial changes needed @P0lip then when you do them, also set the date when you want to get it publish. Publish date is up to you, you can even tell me what hour you want to release it exactly

Reply:
@P0lip will update those changes and we'll aim to publish Wednesday morning!